ReactSkia/components/RSkComponentTextInput.cpp

```diff
@@ -174,6 +195,10 @@ void RSkComponentTextInput::onHandleKey(rnsKey eventKeyType, bool keyRepeat, boo
     textInputMetrics.contentSize.height = paragraph_->getHeight();
     textInputEventEmitter->onFocus(textInputMetrics);
     isInEditingMode_ = true;
+    if(textInputProps.clearTextOnFocus && !displayString_.empty()){
+      displayString_.clear();
+      drawAndSubmit();
```

Comment:
do we need to reset the cursor position as well ?

Reply:
should this clear be under mutex ?

Reply:
need to come up with logic to combine drawAndSubmit call at line 200 and line 203
To avoid double draw calls.{agreed and done with changes}
do we need to reset the cursor position as well ?(yes)
should this clear be under mutex ?(I think it is not need. explained offline)